data_transfer/lib/thinkfast.py

```diff
@@ -73,18 +73,16 @@ def id_in_whitelist(input_ID: str) -> Optional[str]:
     return output_ID
 
 
-def get_participant_id(subjectItems: Dict[int, Any]) -> Optional[str]:
-    # find the ideaFast id in the json...
-    # this is a bit tricky as it's not always in the same place.
+def get_participant_id(subjectItems: dict) -> Optional[str]:
+    # the ID can be placed in any of the list items. Iterate untill found.
     ideaId = ""
```

Comment:
Might be useful to make this None rather than "" so it's explict that if no ID is found that None exists? This does not impact the use of `get_participant_id` below since `if ""` and `if None` both result in same logic.

Reply:
Agreed 👍  - see commit below